src/main/resources/lang/en_us/messages.yml

```
@@ -657,4 +657,16 @@ CRASH_MANAGER_CRASH_RECOVERY_ALERT_DESCRIPTION: |
+ECOMODE_FAILED: "Windows EcoQos may incompatible with your platform: {}, operation cancelled"
+JBR_REQUIRED_TITLE: "Java Runtime Environment Incompatible"
+JBR_REQUIRED_DESCRIPTION: |
+  PeerBanHelper requires JetBrains Runtime (JBR) to run properly on Windows platforms.
+
+  You are currently using: {} {}
+
+  Other Java runtime environments such as Eclipse Adoptium, Azul Zulu, BellSoft Liberca, Oracle/Open JDK may not include patches for issues that PeerBanHelper needs.
+  If you experience crashes related to awt.dll, please switch to JetBrains Runtime or use the PeerBanHelper installer with bundled Java environment.
+  This reminder will only be shown once.
+
+  [Download JetBrains Runtime](https://github.com/JetBrains/JetBrainsRuntime/releases)
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)